### PR TITLE
[DOC] Add Contributing.md

### PR DESCRIPTION
**Pull request by m273d15**
Contribute to #199 

In order to avoid to maintain the same content in `docs/contribute` and in the new file
I  wrote a short summary with (I hope so) the most important information and links to the
extensive description.

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## How to submit changes
+
+### Development Environment
+In order to start with developing Saros you just need:
+
+* a **git** installation and
+* a **JDK 8** installation.
+
+Clone the repository (`git clone https://github.com/saros-project/saros.git`) and buidl Saros via your IDE (more information [here](https://www.saros-project.org/contribute/development-environment.html))
+or via CLI `./gradlew sarosEclipse sarosIntellij sarosServer` (more information [here](https://www.saros-project.org/contribute/development-environment.html#develop-without-an-ide)).
```
[the remark this answers is not in the document]
Comment (m273d15) [reply docs/contributing.md]: It is just my personal taste, but the first thing I search for (in a new project) is the command that is required for building the project, but I am also fine with just referencing the information.

[the remark this answers is not in the document]
Comment (tobous) [reply docs/contributing.md]: If you think it provides good value, keep it in. We then just have to make sure that we also update it if necessary.

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# How to contribute to Saros
+
+Thank you for reading this!
+:tada: We are always searching for new contributors and are delighted by your interest in Saros :tada:
```
Comment (tobous) [inline docs/contributing.md]:
```suggestion
:tada: We are always searching for new contributors and are delighted by your interest in Saros. :tada:
```

File: docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Clone the repository (`git clone https://github.com/saros-project/saros.git`) and buidl Saros via your IDE (more information [here](https://www.saros-project.org/contribute/development-environment.html))
+or via CLI `./gradlew sarosEclipse sarosIntellij sarosServer` (more information [here](https://www.saros-project.org/contribute/development-environment.html#develop-without-an-ide)).
+
+### Development Process
+We use the common [GitHub fork workflow](https://guides.github.com/activities/forking/) and review each pull-request (more information [here](https://www.saros-project.org/contribute/processes/development.html)).
+
+### Guidelines
+In order to speed up the review, please make sure that your pull-request and the commits comply with:
+
+* our [guidelines](https://www.saros-project.org/contribute/guidelines.html).
```
Comment (tobous) [inline docs/contributing.md]:
```suggestion
* our [guidelines](https://www.saros-project.org/contribute/guidelines.html) and
```

File: docs/contributing.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+* the Google Java Format (more information [here](https://www.saros-project.org/contribute/development-environment.html#google-java-format))
```
Comment (tobous) [inline docs/contributing.md]:
```suggestion
* the Google Java Format (more information [here](https://www.saros-project.org/contribute/development-environment.html#google-java-format)).
```

**Pull request merged**